spec/motion-spec/bacon_spec.rb

```diff
@@ -367,11 +367,11 @@ def check(ctx, name)
   end
 
   it 'should work with namespaced modules' do
-    check(Kernel.send(:describe, Motion::Spec::Context) {}, 'Motion::Spec::Context')
+    check(Kernel.send(:describe, MotionSpec::Context) {}, 'MotionSpec::Context')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.